Safety/Src/interchip_S.cpp

```diff
@@ -1,57 +1,42 @@
 #include "spi.h"
 #include "interchip_S.hpp"
 
-static volatile Interchip_Packet rxData;
-static volatile Interchip_Packet txData;
-static volatile bool dataNew;
+static  Interchip_Packet rxData;
+static  Interchip_Packet txData;
+static  bool dataNew;
```

Comment:
Why are these no longer volatile? Is this no longer managed in an interrupt?

Reply:
@AadityaChaudhary

Reply:
They are all still used in interrupts, in my PR I'm pretty sure I made them volatile, not sure why that was removed here.
https://github.com/UWARG/ZeroPilot-SW/blob/d9d68ec1565120f10210cee9dc323109c891d95f/Safety/Src/interchip_S.cpp

Reply:
Okay, I will make em volatile again, then

Reply:
Odd,they are volatile in my local